node/node.go

```diff
@@ -246,7 +246,8 @@ func makeNode(config *cfg.Config,
 		return nil, fmt.Errorf("failed to create peer manager: %w", err)
 	}
 
-	csMetrics, p2pMetrics, memplMetrics, smMetrics := defaultMetricsProvider(config.Instrumentation)(genDoc.ChainID)
+	csMetrics, p2pMetrics, memplMetrics, smMetrics, statesyncMetrics :=
+		defaultMetricsProvider(config.Instrumentation)(genDoc.ChainID)
```

Comment:
nit: this is a code smell to return so many values. We should consider instead returning a struct.

Reply:
sure, will refactor it